README.md

```diff
@@ -1,19 +1,41 @@
-node-red-contrib-tfjs-coco-ssd

```

Comment:
Why change this - it is perfectly good markdown and looks nicer when not read by a markdown editor - (imho)